### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by ghost** [community/community#76746] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I can't find the theme party

**Comment by ghost** [comment]
Below the "Github pages" tab I am not the "THEME CHOOSER" party

**Reply by peterroth** [reply]
Unfortunately the [themes are deprecated](https://github.blog/changelog/2022-08-22-github-pages-deprecating-the-theme-picker/) , this is why you can't find it.

**Comment by theaminulai** [comment]
Regrettably, the [themes ](https://github.blog/changelog/2022-08-22-github-pages-deprecating-the-theme-picker/)have been deprecated, which is the reason for their unavailability.

**Comment by ghost** [comment]
How then to do if in the training that I am he uses the old version?

**Reply by peterroth** [reply]
On the newer version, you can't.
